pkg/appses/ses.go

```diff
@@ -18,15 +19,17 @@ type Config struct {
 type Sender struct {
 	client *ses.SES
 	config Config
+	logger *zap.Logger
```

Comment:
This'll do you in a pinch, but the more idiomatic way would be to pass a context through, and then get the logger off that.